package fiap.cap06aExceptions;

public class TestExceptions4 {

	public static void main(String[] args)  {
		
		Calculadora calc = new Calculadora();
		
		
		System.out.println(calc.dividir(2, 0));
		

	}

}
